action.yml

```diff
@@ -243,6 +245,38 @@ runs:
         CA_GH_ACTION_RESOURCE_GROUP="${{ inputs.containerAppName }}-rg"
         echo "CA_GH_ACTION_RESOURCE_GROUP=${CA_GH_ACTION_RESOURCE_GROUP}" >> $GITHUB_ENV
 
+    - name: Check if the resource group exists
```

Comment:
Should we only check if the resource group exists if the name of the resource group is not set as a parameter? If this parameter is set, we can assume that the customers know what they're doing (as long as the doc is clear) and avoid the extra call.

Reply:
I think that's a fair assumption (as long as it's called out as you mentioned)

Reply:
Thanks!